api-metastore/src/main/java/org/zalando/nakadi/controller/EventTypeController.java

```diff
@@ -90,11 +90,10 @@ public void initBinder(final WebDataBinder binder) {
 
     @RequestMapping(method = RequestMethod.POST)
     public ResponseEntity<?> create(@Valid @RequestBody final EventTypeBase eventType,
-                                    final Errors errors,
-                                    final NativeWebRequest request)
+                                    final Errors errors)
             throws TopicCreationException, InternalNakadiException, NoSuchPartitionStrategyException,
             DuplicatedEventTypeNameException, InvalidEventTypeException, ValidationException,
-            FeatureNotAvailableException {
+            FeatureNotAvailableException, WrongOwningApplicationException {
```

Comment:
Can we name it `InvalidOwningApplicationException` instead?  I believe "wrong" can be interpreted as valid, but not the expected one, though we can't have an expectation here: we don't know better than the user.